.tekton/collector-component-pipeline.yaml

```diff
@@ -101,7 +101,7 @@ spec:
     value: $(tasks.clone-repository.results.commit)
   - description: ""
     name: JAVA_COMMUNITY_DEPENDENCIES
-    value: $(tasks.build-container.results.JAVA_COMMUNITY_DEPENDENCIES)
+    value: $(tasks.build-container-amd64.results.JAVA_COMMUNITY_DEPENDENCIES)
 
```

Comment:
[not about this line] Please don't forget to adjust the comment in the `init` task: we don't use `apply-tags` any more but rather retag via building a manifest.